ui/selectors/transactions.js

```diff
@@ -20,6 +21,12 @@ const INVALID_INITIAL_TRANSACTION_TYPES = [
   TransactionType.cancel,
   TransactionType.retry,
 ];
+const allowedSwapsSmartTransactionStatusesForActivityList = [
+  SmartTransactionStatuses.PENDING,
+  SmartTransactionStatuses.UNKNOWN,
+  SmartTransactionStatuses.RESOLVED,
+  SmartTransactionStatuses.CANCELLED,
```

Comment:
Hey Dan! Could you include a comment explaining why this particular subset of statuses was chosen? In particular, why we omit the `SUCCESS` and all the `CANCELLED_X` statuses?